builder/qemu/builder_test.go

```diff
@@ -302,6 +302,24 @@ func TestBuilderPrepare_UseBackingFile(t *testing.T) {
 	}
 }
 
+func TestBuilderPrepare_SkipResizeDisk(t *testing.T) {
+	var b Builder
+	config := testConfig()
+
+	config["skip_resize_disk"] = true
+
+	// Bad: iso_url is not a disk_image
+	config["disk_image"] = false
+	b = Builder{}
+	_, warns, err := b.Prepare(config)
+	if len(warns) > 0 {
+		t.Fatalf("bad: %#v", warns)
+	}
+	if err == nil {
+		t.Fatal("should have error")
+	}
+}
+
```

Comment:
At first glance it is not clear what is being tested here. But it seems like this is a mutual exclusion test to ensure that Prepare errors when "skip_resize_disk" is true, and disk image is not set. Including a test case to ensure the good state does not error would also be helpful. 

```suggestion
func TestBuilderPrepare_SkipResizeDisk(t *testing.T) {
	config := testConfig()
	config["skip_resize_disk"] = true
	config["disk_image"] = false
	
	b := Builder{}
	_, warns, err := b.Prepare(config)
	if len(warns) > 0 {
		t.Errorf("unexpected warns when calling prepare with skip_resize_disk set to true: %#v", warns)
	}
	if err == nil {
	        t.Errorf("setting skip_resize_disk to true when disk_image is false should have error")
	}
}

```